### ISSUE TITLE
MMCV-full version issue

### ISSUE THREAD

**Issue by yyyyyyfs** [fcjian/PromptDet#13]
when try to install mmdet==2.16.0., It will be automatically installed mmcv-full==1.7.1.  It‘s impossible to satisfy mmcv-full<1.4.0.
Can you release all your version information in your conda environment？

**Comment by fcjian** [comment]
Package                Version         Location
---------------------- --------------- -------------------------------------------------------------------------------------------------------------------------
addict                 2.4.0
albumentations         1.2.1
appdirs                1.4.4
argon2-cffi            21.3.0
argon2-cffi-bindings   21.2.0
asynctest              0.13.0
attributee             0.1.5
attrs                  21.4.0
backcall               0.2.0
beautifulsoup4         4.11.1
bleach                 5.0.0
braceexpand            0.1.7
cached-property        1.5.2
certifi                2021.10.8
cffi                   1.15.0
chardet                4.0.0
charset-normalizer     2.0.12
cityscapesScripts      2.2.0
click                  7.1.2
clip                   1.0
codecov                2.1.12
colorama               0.4.4
coloredlogs            15.0.1
coverage               6.3.2
cycler                 0.11.0
Cython                 0.29.28
dataclasses            0.6
debugpy                1.6.0
decorator              5.1.1
defusedxml             0.7.1
docker-pycreds         0.4.0
docopt                 0.6.2
dotty-dict             1.3.0
entrypoints            0.4
ExifRead               3.0.0
faiss-cpu              1.7.2
fastjsonschema         2.15.3
fire                   0.4.0
flake8                 4.0.1
flatbuffers            2.0
fonttools              4.33.3
ftfy                   6.1.1
gitdb                  4.0.9
GitPython              3.1.27
h5py                   3.6.0
hope                   3.5.14
humanfriendly          10.0
idna                   2.10
imagecorruptions       1.1.2
imageio                2.19.1
img2dataset            1.12.0
importlib-metadata     4.2.0
importlib-resources    5.7.1
iniconfig              1.1.1
interrogate            1.5.0
ipykernel              6.13.0
ipython                7.33.0
ipython-genutils       0.2.0
ipywidgets             7.7.0
isort                  4.3.21
jedi                   0.18.1
Jinja2                 3.1.2
joblib                 1.1.0
jsonschema             4.5.1
jupyter                1.0.0
jupyter-client         7.3.1
jupyter-console        6.4.3
jupyter-core           4.10.0
jupyterlab-pygments    0.2.2
jupyterlab-widgets     1.1.0
kiwisolver             1.4.2
kwarray                0.6.0
lap                    0.4.0
lvis                   0.5.3
MarkupSafe             2.1.1
matplotlib             3.5.2
matplotlib-inline      0.1.3
mccabe                 0.6.1
mistune                0.8.4
mkl-fft                1.3.1
mkl-random             1.2.2
mkl-service            2.4.0
mmcls                  0.23.0
mmcv-full              1.3.16
mmdet                  2.16.0
mmtrack                0.13.0
motmetrics             1.2.5
mpmath                 1.2.1
mt-shop                0.0.5
nbclient               0.6.3
nbconvert              6.5.0
nbformat               5.4.0
nest-asyncio           1.5.5
networkx               2.6.3
notebook               6.4.11
numpy                  1.21.5
onnx                   1.7.0
onnxruntime            1.11.1
opencv-python          4.5.5.64
opencv-python-headless 4.6.0.66
ordered-set            4.1.0
packaging              21.3
pandas                 1.3.5
pandocfilters          1.5.0
parso                  0.8.3
pathtools              0.1.2
pexpect                4.8.0
pickleshare            0.7.5
Pillow                 9.0.1
pip                    21.2.4
pluggy                 1.0.0
prettytable            1.0.1
prometheus-client      0.14.1
promise                2.3
prompt-toolkit         3.0.29
protobuf               3.20.1
psutil                 5.9.0
ptyprocess             0.7.0
py                     1.11.0
pyarrow                8.0.0
pycocotools            2.0.2
pycodestyle            2.8.0
pycparser              2.21
pyflakes               2.4.0
Pygments               2.12.0
pyparsing              3.0.8
pyquaternion           0.9.9
pyrsistent             0.18.1
pytest                 7.1.2
python-dateutil        2.8.2
pytz                   2022.1
PyWavelets             1.3.0
PyYAML                 6.0
pyzmq                  22.3.0
qtconsole              5.3.0
QtPy                   2.1.0
qudida                 0.0.4
regex                  2022.4.24
requests               2.25.1
retrying               1.3.3
scikit-image           0.19.2
scikit-learn           1.0.2
scipy                  1.7.3
seaborn                0.11.2
Send2Trash             1.8.0
sentry-sdk             1.7.2
setproctitle           1.2.3
setuptools             57.4.0
setuptools-scm         6.4.2
shortuuid              1.0.9
six                    1.16.0
sklearn                0.0
smmap                  5.0.0
soupsieve              2.3.2.post1
sympy                  1.10.1
tabulate               0.8.9
termcolor              1.1.0
terminado              0.13.3
terminaltables         3.1.10
threadpoolctl          3.1.0
tifffile               2021.11.2
tinycss2               1.1.1
toml                   0.10.2
tomli                  2.0.1
torch                  1.8.0
torchaudio             0.8.0a0+a751e1d
torchvision            0.9.0
tornado                6.1
tqdm                   4.60.0
traitlets              5.1.1
typing                 3.7.4.3
typing-extensions      3.10.0.0
ubelt                  1.0.1
urllib3                1.26.9
wandb                  0.12.21
wcwidth                0.2.5
webdataset             0.2.5
webencodings           0.5.1
wheel                  0.37.0
widgetsnbextension     3.6.0
xdoctest               1.0.0
xmltodict              0.13.0
yapf                   0.32.0
zipp                   3.8.0

**Comment by yyyyyyfs** [comment]
Thanks for youe reply！ I have solved this problem by replace the CUDA version. I found that mmcv-full==1.3.8 only suitable the CUDA version<=11.0 ( I tried CUDA version = 10.2, 11.0, 11.1, 11.3, and the version of 10.2 and 11.0 worked.), and mmdet==2.16.0, torch==1.8.1.
